File: SDKs/amazonaws.com/appflow/2020-08-23/python/src/sdk/models/shared/redshiftconnectorprofilecredentials.py
from dataclasses import dataclass, field
from dataclasses_json import dataclass_json


@dataclass_json
@dataclass
class RedshiftConnectorProfileCredentials:
    password: str = field(default=None, metadata={'dataclasses_json': { 'field_name': 'password' }})
    username: str = field(default=None, metadata={'dataclasses_json': { 'field_name': 'username' }})
    

File: SDKs/amazonaws.com/appflow/2020-08-23/typescript/src/sdk/models/shared/redshiftconnectorprofilecredentials.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// RedshiftConnectorProfileCredentials
/** 
 *  The connector-specific profile credentials required when using Amazon Redshift. 
**/
export class RedshiftConnectorProfileCredentials extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=password" })
  password: string;

  @SpeakeasyMetadata({ data: "json, name=username" })
  username: string;
}
